wlm/src/main/java/ai/djl/serving/wlm/LmiConfigRecommender.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package ai.djl.serving.wlm;
+
+import ai.djl.util.Utils;
+import ai.djl.util.cuda.CudaUtils;
+
+import java.util.Map;
+import java.util.Properties;
+
+/** A utility class to auto configure LMI model properties. */
+public final class LmiConfigRecommender {
+
+    private static final Map<String, String> MODEL_TO_ROLLING_BATCH =
+            Map.ofEntries(
+                    Map.entry("falcon", "lmi-dist"),
```

Comment:
falcon can also run on vllm well. We can revisit this once Rubikon Engine start to come